app/models/commConns.js

```diff
@@ -19,6 +33,35 @@ class CommConns {
       }).catch(reject);
     })
   }
+
+  static findByCommId (communicationId) {
```

Comment:
added `findManyByAttribute` but yes as we discussed need to also be able to add the whole  logic that supports `.and.where("retired", null)`